<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="menu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Главное меню</string>
    </property>
   </widget>
   <widget class="QLabel" name="text">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>661</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Текст вопроса</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>401</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="v1">
       <property name="text">
        <string>Ответ 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="v2">
       <property name="text">
        <string>Ответ 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="v3">
       <property name="text">
        <string>Ответ 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="v4">
       <property name="text">
        <string>Ответ 4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="nextquestion">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Далее</string>
    </property>
   </widget>
   <widget class="QLabel" name="gotright">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="qnumber">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вопрос</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endtest2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Завершить тест</string>
    </property>
   </widget>
   <widget class="QLabel" name="bck">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>bck</zorder>
   <zorder>menu</zorder>
   <zorder>text</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>nextquestion</zorder>
   <zorder>gotright</zorder>
   <zorder>qnumber</zorder>
   <zorder>endtest2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
